### DISCUSSION TITLE
通用设置>显示>显示通知区图标，取消勾选后，会重新被勾选

### DISCUSSION THREAD

**Discussion by lxl217114** [zhongyang219/TrafficMonitor#1712] [General]
显示通知区图标，取消勾选后，会重新被勾选
如下图设置，取消勾选后，重启/时间比较长以后，又会重新显示图标
![image](https://github.com/zhongyang219/TrafficMonitor/assets/54263659/a6bf46cb-4675-4c18-b738-926daf31fea9)

**Comment by zhongyang219** [comment]
这个问题已经解决了，请等待更新
